project.scala

```diff
@@ -2,3 +2,14 @@
 //> using platform scala-native
 //> using nativeVersion 0.5.8
 //> using scalacOptions -Wconf:msg=indented:silent
+
+// Ignore the sandbox folder
+//> using exclude sandbox/*
+
+// Made up publishing values!!
+// More settings: https://scala-cli.virtuslab.org/docs/reference/directives#publish
+// See publish docs: https://scala-cli.virtuslab.org/docs/commands/publishing/publish-setup
+
+//> using publish.organization com.finochiom
+//> using publish.name s2d
+//> using publish.version 0.0.1
```

Comment:
If you do: `scala-cli --power publish local .`

Your lib with be publish to your local machine, as in the screenshot, to your local `.ivy2` folder in your home directory.

![image](https://github.com/user-attachments/assets/795bb894-01a9-414d-8111-d2b09da29e22)

Obviously I've just made up some values. 😅 
Your organisation, for example, should be a domain that you own in some way.